docs/production-deployment/cloud/how-to-manage-audit-logging.md

```diff
@@ -63,6 +63,16 @@ The following list specifies both the supported events and the Temporal APIs tha
   - Rename custom Search Attribute: `RenameCustomSearchAttribute`
   - Request increase in Retention Period: `UpdateNamespace`
 
+### API Key Operation events
+
+The following list specifies both the supported events and the Temporal APIs that emit the logs for API Key Operations:
+
+- **Create API Key**: Creates an API Key.
```

Comment:
nit - the format of the API Key commands are different than the format for https://temporal-documentation-git-api-key.preview.thundergun.io/cloud/audit-logging#admin-operation-events .  May we make the formats match, please?